packages/lesswrong/lib/collections/moderatorActions/helpers.ts

```diff
@@ -1,5 +1,42 @@
 import moment from "moment";
 import { forumTypeSetting } from "../../instanceSettings";
+import ModeratorActions from "./collection";
+import { rateLimits, RateLimitType, RATE_LIMIT_ONE_PER_DAY, RATE_LIMIT_ONE_PER_FORTNIGHT, RATE_LIMIT_ONE_PER_MONTH, RATE_LIMIT_ONE_PER_THREE_DAYS, RATE_LIMIT_ONE_PER_WEEK } from "./schema";
+
+/**
+ * For a given RateLimitType, returns the number of hours a user has to wait before posting again.
+ */
+export function getTimeframeForRateLimit(type: RateLimitType) {
+  let hours 
+  switch(type) {
+    case RATE_LIMIT_ONE_PER_DAY:
+      hours = 24; break;
+    case RATE_LIMIT_ONE_PER_THREE_DAYS:
+      hours = 24 * 3; break;
+    case RATE_LIMIT_ONE_PER_WEEK:
+      hours = 24 * 7; break;
+    case RATE_LIMIT_ONE_PER_FORTNIGHT:
+      hours = 24 * 14; break;
+    case RATE_LIMIT_ONE_PER_MONTH:
+      hours = 24 * 30
```

Comment:
nit: prefer `break;` on a new line, also `break;` after the last case (makes future bugs with e.g. that case falling through if we ever add a new case below it much less likely)